values.yaml

```
@@ -202,6 +202,13 @@ injector:
     # Extra annotations to attach to the injector service
     annotations: {}
 
+  # strategy for updating the deployment. This can be a multi-line string or a YAML map.
+  strategy: |
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+
```

What do you think about defaulting to an empty value, instead of an explicit default? 

From looking through the values file, it looks like we've tended to default to YAML for options that support both YAML and multi-line string formats. Something like this?

```suggestion
  # strategy for updating the deployment. This can be a multi-line string or a YAML map.
  strategy: {}
  # strategy: |
  #   rollingUpdate:
  #     maxSurge: 25%
  #     maxUnavailable: 25%
  #   type: RollingUpdate
```